eng/pipelines/templates/jobs/ci.yml

```diff
@@ -233,10 +235,22 @@ jobs:
       MatrixFilters: ${{ parameters.MatrixFilters }}
       MatrixReplace: ${{ parameters.MatrixReplace }}
       CloudConfig:
-        Cloud: Public
+        Cloud: public
+      ${{ if ne(parameters.TestDependsOnDependency, 'not-specified') }}:
+        ProjectListOverrideFileProperty: 'ProjectListOverrideFileProperty'
+        SparseCheckoutPaths: 
+          - "/sdk"
+          - "/tools/bootstrapTools/bootstrap.targets"
+          - "!/sdk/*/**/SessionRecords"
+          - "common/Perf/Azure.Test.Perf/Azure.Test.Perf.csproj"
+        PreGenerationSteps: 
+        - template: /eng/pipelines/templates/steps/dependency.tests.yml
+          parameters:
```

Comment:
We might also want to pass in the ServiceDirectory so that we can exclude any tests from the serviceDirectory because we will be running those on the normal test runs and not need to run them again on the dependency test runs.

Reply:
Done